py/torch_tensorrt/dynamo/backend/backends.py

```diff
@@ -28,7 +29,24 @@ def torch_tensorrt_backend(
     gm: torch.fx.GraphModule,
     sample_inputs: Sequence[torch.Tensor],
     settings: CompilationSettings = CompilationSettings(),
+    **kwargs
 ):
+    # If the user specifies keyword args, overwrite those fields in settings
+    # Validate all specified kwargs to ensure they are true fields of the dataclass
+    #
+    # Note: kwargs provided by torch.compile are wrapped in the "options" key
+    if kwargs:
```

Comment:
Why do we have settings if they are just going to be overwritten by kwargs? should we just parse kwargs internally?

Reply:
My intent here was to _allow_ overwrite of settings with `kwargs`, but to also allow users to specify a `settings` object directly if instead preferred. The main need for `kwargs` parsing is `torch.compile`'s new `options` passing to the backend, for which we need to allow `kwargs`.